### DISCUSSION TITLE
Recommended settings for export & printing?

### DISCUSSION THREAD

**Discussion by cknotz** [koenderks/aRtsy#163] [Q&A]
Dear Koen,

thanks for this package, great work!

I was wondering: If I want to export an image and print it out on a larger canvas, etc. -- what settings (resolution, dimensions,...) should I ideally use so that the final product looks decent? Do you have any experiences or recommendations?

Best
Carlo

**Comment by koenderks** [comment]
Hi Carlo,

Thanks for the nice words. For printing, I would recommend exporting the image to a vector format (i.e., .pdf or .svg) with dimensions of your choice (I found 50 cm by 50 cm to look nice in a similar sized frame).

So, something like this would do the trick!

```r
artwork <- canvas_flame(colors = c("black", "white"))
saveCanvas(artwork, filename = "myArtwork.pdf", height = 50, width = 50)
``` 

The nice thing about the vector formats .svg and .pdf is that they keep their quality regardless of the size in which you save them.

Of course, you should check the final file before the printing, but I think this should work, I have printed one with these dimensions as well. If the height, width = 50 settings strech out the image too much in the pdf file you can also try the default parameters of 7, I guess it will be a little bit try and error to see which dimensions give the nicest pdf.

Let me know what your experiences are with this! :)

Koen
